packages/backend-ai/src/get_message.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  BedrockRuntimeClient,
+  ConverseCommand,
+  ConverseCommandInput,
+  ConverseCommandOutput,
+  Message,
+} from '@aws-sdk/client-bedrock-runtime';
+import {
+  ContentBlock,
+  GetMessageInput,
+  GetMessageOutput,
+  Tool,
+} from './types.js';
+
+/**
+ * Sends a message to the AI model and retrieves a response.
+ * This function leverages ConverseCommand (https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/bedrock-runtime/command/ConverseCommand/)
+ * @param input - The input parameters for the function.
+ * @param input.modelId - The ID of the AI model to use for generating a response.
+ * @param input.messages - The list of message objects to send to the AI model.
+ * @param input.systemPrompts - The list of system prompts to include with the request.
+ * @param input.tools - The list of optional tools to use for generating a response.
+ * @returns The response from the AI model.
+ * @throws Eror If the response from the AI model does not contain a message.
+ */
+export const getMessage = async ({
```

Comment:
How are customers expected to consume this function? Do they import it in their lambda function code? Could we just expose some prefilled `defineFunction` definitions that they could use directly and not have to touch lambda code at all?

Reply:
I believe we expect to have a lot of customizable within the lambda execution.